src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3739,6 +3739,11 @@ public partial class ExternalException : System.SystemException
         public virtual int ErrorCode { get; }
         public override string ToString() { throw null; }
     }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(2048), Inherited = false)]
+    public sealed partial class InAttribute : System.Attribute
+    {
+        public InAttribute() { }
```

Comment:
Nit: The layout of this file is not super consistent ... but it would be a bit more logical to add the InAttribute to the same block / next to OutAttribute.

Reply:
You may want to submit this move in separate PR, so the PR with the Span changes is smaller.

Reply:
> it would be a bit more logical to add the InAttribute to the same block / next to OutAttribute.

Was following alphabetical order. I will update it.

Reply:
https://github.com/dotnet/corefx/pull/24933